examples/use_openai_assistant_json.rs

```diff
@@ -0,0 +1,115 @@
+use std::ffi::OsStr;
+use std::path::Path;
+
+use allms::assistants::{OpenAIAssistant, OpenAIAssistantVersion, OpenAIFile, OpenAIVectorStore};
+use allms::llm_models::OpenAIModels;
+
+use anyhow::{anyhow, Result};
+
+const CONCERT_INFO_SCHEMA: &str = r#"
```

Comment:
This example is mostly identical, but instead of passing a struct for type checking we pass a Json schema.